src/test_l1_hdmi_cec_driver.c

```diff
@@ -2231,16 +2239,33 @@ static UT_test_suite_t *pSuite_panel = NULL;
  */
 int test_hdmicec_hal_l1_register( void )
 {
-
+    ut_kvp_status_t status;
+    char    deviceType[TEST_UTIL_DEVICE_TYPE_SIZE];
+    // Reading Extended enum support form profile file
+    extendedEnumsSupported = ut_kvp_getBoolField( ut_kvp_profile_getInstance(), "hdmicec/features/extendedEnumsSupported" );
+    // Getting device type from profile.
+    status = ut_kvp_getStringField(ut_kvp_profile_getInstance(), "hdmicec/Type", deviceType, TEST_UTIL_DEVICE_TYPE_SIZE);
```

Comment:
there's a macro for bool, string, uint etc. 

- `UT_KVP_PROFILE_GET_STRING` 
- `UT_ASSERT_EQUAL_KVP_PROFILE_BOOL`

`extendedEnumsSupported = UT_ASSERT_EQUAL_KVP_PROFILE_BOOL( "hdmicec/features/extendedEnumsSupported" );`

Reply:
Added Macro instead of the direct API call,

But in main.c  the API is already being called directly, We need to stick to one method for the uniformity of the code.